src/config.ts

```diff
@@ -116,33 +119,11 @@ const EXCLUDED_SOURCES = (() => {
     }
 })();
 
-const gasSchedule: { [key in ERC20BridgeSource]: number } = {
-    [ERC20BridgeSource.Native]: 1.5e5,
-    [ERC20BridgeSource.Uniswap]: 3e5,
-    [ERC20BridgeSource.LiquidityProvider]: 4.5e5,
-    [ERC20BridgeSource.Eth2Dai]: 5.5e5,
-    [ERC20BridgeSource.Kyber]: 8e5,
-    [ERC20BridgeSource.CurveUsdcDai]: 9e5,
-    [ERC20BridgeSource.CurveUsdcDaiUsdt]: 9e5,
-    [ERC20BridgeSource.CurveUsdcDaiUsdtTusd]: 10e5,
-    [ERC20BridgeSource.CurveUsdcDaiUsdtBusd]: 10e5,
-};
-
-const feeSchedule: { [key in ERC20BridgeSource]: BigNumber } = Object.assign(
-    {},
-    ...(Object.keys(gasSchedule) as ERC20BridgeSource[]).map(k => ({
-        [k]: new BigNumber(gasSchedule[k] + 1.5e5),
-    })),
-);
-
 export const ASSET_SWAPPER_MARKET_ORDERS_OPTS: Partial<SwapQuoteRequestOpts> = {
     excludedSources: EXCLUDED_SOURCES,
     bridgeSlippage: DEFAULT_QUOTE_SLIPPAGE_PERCENTAGE,
-    maxFallbackSlippage: DEFAULT_FALLBACK_SLIPPAGE_PERCENTAGE,
```

Comment:
Is this related to changes in Asset-swapper?

Reply:
yup, that's in https://github.com/0xProject/0x-api/pull/162/commits/5f8381d43f7fa58a8decf803c017b91da13615e5